packages/internal-workflow-common/src/retry-policy.ts

```diff
@@ -71,3 +71,19 @@ export function compileRetryPolicy(retryPolicy: RetryPolicy): temporal.api.commo
     nonRetryableErrorTypes: retryPolicy.nonRetryableErrorTypes,
   };
 }
+
+export function uncompileRetryPolicy(
```

Comment:
uncompile is a bit of a weird term here. If you wanna stay consistent with above, `decompile` is probably more common

Alternatively, `extract`